src/pkg/clouds/aws/ecs/cfn/setup.go

```diff
@@ -185,13 +175,12 @@ func (a *AwsEcs) fillWithOutputs(dso *cloudformation.DescribeStacksOutput) error
 	for _, output := range dso.Stacks[0].Outputs {
 		switch *output.OutputKey {
 		case outputs.SubnetID:
+			// Only set the SubNetID if it's not already set; this allows the user to override the subnet
 			if a.SubNetID == "" {
 				a.SubNetID = *output.OutputValue
 			}
 		case outputs.TaskDefArn:
-			if a.TaskDefARN == "" {
-				a.TaskDefARN = *output.OutputValue
-			}
+			a.TaskDefARN = *output.OutputValue
```

Comment:
Interesting. Why did we have this `a.TaskDefARN == ""` guard in the first place?

Reply:
No clue in GIT: it's part of the first commit from Dec 27, 2023.